docs/multiindex_isotope_decay_data.ipynb

```diff
@@ -0,0 +1,446 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    config = config_reader.Configuration.from_yaml("/Users/anirbandutta/Projects/gamma_ray_tardis/tardis_configv1_density_exponential_nebular.yml")
</pre><p>See relative path mention (this config should be in tardis already)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>